CHANGELOG.md

```diff
@@ -1,3 +1,292 @@
+  $ASSETS_DIR/terraform-modules/$PLATFORM/flatcar-linux/kubernetes/workers/require.tf \
+  $ASSETS_DIR/terraform-modules/dns/route53/require.tf 2> /dev/null
+  ```
+
+4. Go to the `terraform` directory:
+  ```sh
+  cd $ASSETS_DIR/terraform
+  ```
+
+5. Replace the old providers:
+  ```sh
+  terraform state replace-provider -auto-approve registry.terraform.io/-/ct registry.terraform.io/poseidon/ct && \
+  terraform state replace-provider -auto-approve registry.terraform.io/-/template registry.terraform.io/hashicorp/template
+  ```
+
+6. **NOTE:** On clusters with a single controller node only, delete the old `kube-apiserver` ReplicaSet:
+  ```sh
+  kubectl delete rs -n kube-system $(kubectl get rs -n kube-system -l k8s-app=kube-apiserver --no-headers=true  | sort -k5 -h | head -n +2 | awk '{print $1}')
+  ```
+
+7. `FelixConfiguration` has been moved to calico charts. To avoid firewall blip, label and annotate it, so that it can be managed by Helm while updating:
+  ```sh
+  kubectl label FelixConfiguration default app.kubernetes.io/managed-by=Helm --overwrite=true && \
+  kubectl annotate FelixConfiguration default meta.helm.sh/release-name=calico --overwrite=true && \
+  kubectl annotate FelixConfiguration default meta.helm.sh/release-namespace=kube-system --overwrite=true
+  ```
+
+Finally, run the following:
+
+```sh
+lokoctl cluster apply --skip-components -v
+```
+
+The update process typically takes about 10 minutes.
+After the update, running `lokoctl health` should result in an output similar to the following:
+
+```sh
+Node                     Ready    Reason          Message
+
+lokomotive-controller-0  True     KubeletReady    kubelet is posting ready status
+lokomotive-1-worker-0    True     KubeletReady    kubelet is posting ready status
+lokomotive-1-worker-1    True     KubeletReady    kubelet is posting ready status
+lokomotive-1-worker-2    True     KubeletReady    kubelet is posting ready status
+Name      Status    Message              Error
+
+etcd-0    True      {"health":"true"}
+```
+
+#### Updating native kubelets and etcd (optional)
+
+- Manually update etcd following the steps mentioned in the doc
+  [here](https://github.com/kinvolk/lokomotive/blob/v0.5.0/docs/how-to-guides/upgrade-etcd.md).
+- Manually update the kubelet running on the nodes, by following the steps mentioned in the doc
+  [here](https://github.com/kinvolk/lokomotive/blob/v0.5.0/docs/how-to-guides/upgrade-bootstrap-kubelet.md).
+
+#### Updating cert-manager
+
+Run the following command until it exits with code zero:
+
+```sh
+lokoctl component render-manifest cert-manager | kubectl apply -f -
+```
+
+Now it is safe to update:
+
+```sh
+lokoctl component apply cert-manager
+```
+
+#### Updating prometheus-operator
+
+Delete `psp prometheus-operator-admission` as it does not apply annotation `runtime/default` while updating:
+
+```sh
+kubectl delete psp prometheus-operator-admission
```

Comment:
I confirmed, that this solves the problem. The weird thing is that on `v0.4.1` cluster the PSP is there, but shouldn't as Helm hook should remove it.